docs/en/index.md

```diff
@@ -71,29 +88,25 @@ class SendNewslettersCommand extends Command
 }
 ```
 
-The configure method is to name the command and specify arguments and options.
-They have a lot of options and you can read about them in Symfony Documentation.
 
-When the command is executed, the execute method is called with two arguments.
-First one is command input, which contains all the parsed arguments and options.
-The second one is command output which should be used to provide feedback to the developer which ran the command.
+In `__construct`, we setup the name of the command, then set some description and set all dependencies.
```

Comment:
I disagree about the configure method https://github.com/Kdyby/Console/pull/50#issuecomment-182394814.

Anyway, this PR is tryting to solve too many issues at once, making it hart to discuss them all at once and come up with a solution :-/